backend/main.py

```diff
@@ -88,59 +100,44 @@ async def retrieve_available_lights():
 @fast_app.post("/selected-lights")
 def store_selected_lights(lights: List[str]):
     if not lights:
-        raise HTTPException(status_code=400, detail="Selected lights list is required")
+        raise HTTPException(status_code=400, detail='\"lights\" list is required.')
 
     try:
-        redis_client.set("lights", ",".join(lights))
+        redis_client.set(constants.REDIS_SELECTED_LIGHTS_KEY, ",".join(lights))
 
-        response = StandardResponse(
+        return StandardResponse(
             success=True, message="Selected lights list stored in Redis"
         )
 
     except redis.exceptions.RedisError as redis_err:
         raise HTTPException(status_code=500, detail=f"Redis Error: {str(redis_err)}")
 
-    return response
-
 
 @fast_app.put("/start-spotihue")
 async def start_spotihue(lights: List[str] = None):
-    try:
-        if not lights:
-            # stored_lights = redis_client.get("lights")
-            # if stored_lights:
-            #     lights = stored_lights.split(",")
```

Comment:
So my plan was to store the lights as a comma separated list in Redis since we can't store a list, but then split on that string to create a list that you can then pass/iterate over... What are your thoughts on how we handle storing the selected lights?

Reply:
The more I think about it, I don't think we need to store selected lights at any point. 

For start-spotihue: it accepts a list of lights from the request rather than reading cached lights, which I think is best; the frontend can know which lights are selected and which are un-selected in the UI at any given point in time, and when the "start spotihue" button is clicked, it can just pass that list of selected light names to the backend.

For stop-spotihue: this is stopping the currently-running task, which has already been initialized with the list of lights originally passed to start-spotihue, and no user input can change that task object's list of lights it has been working with. If we want to not only stop the task, but also change the lights back to white we could either: 
        * (if we want only change the task's lights to white) access that task object's .args on the backend and get the lights to be changed that way, OR
        * (if we want to change ALL available lights to white, no matter what lights the task has been working with) get the hue bridge's reachable lights.

Thoughts?

Reply:
This option:
* (if we want only change the task's lights to white) access that task object's .args on the backend and get the lights to be changed that way,